src/runtime/print.go

```diff
@@ -349,7 +349,7 @@ func printmap(m *hashmap) {
 
 func printptr(ptr uintptr) {
 	if ptr == 0 {
-		print("nil")
+		print("0x0")
```

Comment:
Any reason for this change? I know it didn't match upstream exactly, but `nil` seems fine to me.

Reply:
To match upstream. I will revert it.